gnuboardProject/frontend/src/components/Profile/Profile.tsx

```
@@ -127,40 +245,84 @@ const Profile: React.FC = () => {
+    const { name, value } = e.target;
+    setEditProfile(prev => ({ ...prev, [name]: value }));
+  };
+
+  const handleImgChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      setEditProfile(prev => ({ ...prev, profileImage: e.target.files![0] as File }));
+      setImgPreview(URL.createObjectURL(e.target.files[0]));
+    }
+  };
+
+  const handlePhoneEditChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)